plugins/modules/aci_export_encryption.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_export_encryption
+short_description: Manage Global AES Passphrase Encryption Settings (pki:ExportEncryptionKey)
+description:
+- Manage Global AES Passphrase Encryption Settings on Cisco ACI fabrics.
+options:
+  passphrase:
+    description:
+    - The AES passphrase to use for configuration export encryption.
+    - This cannot be modified once in place on the APIC. To modify an existing passphrase, you must delete it by sending a request with state C(absent).
```

Comment:
We could, but this would prevent changing the value of strong_encryption for an existing passphrase without deleting and re-adding the AES passphrase.

Reply:
ok